packages/ui/src/components/search/MessageResultDriver.tsx

```diff
@@ -0,0 +1,8 @@
+//
+export const MessageResultDriver = () => {
```

Comment:
what is message result? if it is search results lets uniform that all over the place to only use search results

Message result page is the page you view after clicking a search result
Search results is the search results